rpgJavaInterpreter-core/src/test/resources/db/T53_A01.rpgle

```diff
@@ -0,0 +1,28 @@
+     FEMPLOYEE  IF   E           K DISK    UsrOpn
```

Comment:
Change name and remove any reference to `£DBG_Pas`  `£DBG_Str` and simplify the test because these variables are only noise.

The tests with naming convention `Txx_Ayy.rpgle` extracted from `MULANGT90` must be placed in `resources/smeup`, and in that case is helpful to use `£DBG_Pas` and  `£DBG_Str`

Reply:
Done